src/components/internalSettings/InternalSettings.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-underscore-dangle */
+import React from "react";
+import { Accordion, Stack } from "@chakra-ui/react";
+import { useParams } from "react-router-dom";
+import useAxios from "axios-hooks";
+
+import AccordionSection from "./AccordionSection";
+import Loading from "../../util/Loading";
+
+const InternalSettings: React.FC = () => {
+  const { hexathonId } = useParams();
+
+  const [{ data: applications, loading, error }] = useAxios(
+    `https://registration.api.hexlabs.org/applications/`
+  );
```

Comment:
So the internal settings data should pull from https://docs.hexlabs.org/#/registration/get_branches instead of the endpoint listed. You are also able to provide a `hexathonId` as a query param as specified in the documentation. The way that you are currently doing it, you'll have a new accordion section for each application versus each branch.